tutorials/mnist_tutorial_tf.py

```diff
@@ -24,11 +24,18 @@
 flags.DEFINE_float('learning_rate', 0.1, 'Learning rate for training')
 
 
-def main(argv=None):
+def mnist_tutorial(testing=False):
```

Comment:
aesthetically, I think it would be nicest not to have `testing` flags, but just have general problem size flags. That way the test can call the tutorial with very small problem sizes, and the user is also free to shrink the problem size if they want it to run faster for whatever reason.